recipe/meta.yaml

```diff
@@ -52,6 +52,7 @@ outputs:
         - xorg-libxrender    # [linux]
         - xorg-libx11        # [linux]
         - xorg-libxmu        # [linux]
+        - cairo
```

Comment:
The bug arises because of the linux container, but cairo should be installed on all platforms?

Reply:
Wouldn't it be beneficial to have cairo support on all platforms?

Reply:
Probably; I don't know. The build succeeds... It just caught my eye because everything above it was limited to `[linux]`.